README.md

```diff
@@ -37,3 +38,17 @@ npm install -g webpack
 npm install
 ```
 
+### Step 2: Install MongoDB
+Follow the instructions for your set up at https://www.mongodb.com/try
+The database name is fec
+
```

Comment:
If I am not mistaken, your `npm install` installs MongoDB.

Reply:
I'm not really sure about it. Npm installs mongoose, the ORM for Mongo. When I went to the mongoose docs, part of their installation instructions are to first install MongoDB, then run npm install mongoose. So I think they might be separate?